compiler.ts

```diff
@@ -239,7 +243,7 @@ function codeGenAssignable(target: Assignable<Type>, value: string[], env: Globa
     default:
       // Force type error if assignable is added without implementation
       // At the very least, there should be a stub
-      const err: never = target;
+      const err: never = <never>target;
```

Comment:
Note for future:
This change invalidated the entire check that was being done here. doing this forcibly typecast the target to `never`, avoiding the entire point of the check requiring a stub for `bracket-lookup` not being implemented. It should have instead been resolved with a `case "bracket-lookup": throw new Error("Assignment to bracket lookup not yet supported")`